include/swift/AST/DiagnosticsSema.def

```diff
@@ -667,6 +667,9 @@ ERROR(reserved_member_name,none,
       " 'foo.%1' expression", (DeclName, StringRef))
 
 ERROR(invalid_redecl,none,"invalid redeclaration of %0", (DeclName))
+ERROR(invalid_redecl_synth_init,none,
+      "invalid redeclaration of synthesized "
+      "%select{default|memberwise}1 %0", (DeclName, bool))
```

Comment:
Do we actually use the term "default initializer" for a no-argument initializer? Unlike in C++, it's not the "default" if you don't initialize something. It might be fine just to leave out the qualifier in that case.

Reply:
OK. I tend to call "default" an synthesized no-argument initializer.

Reply:
[Heh](https://docs.swift.org/swift-book/LanguageGuide/Initialization.html#ID213) :)

Reply:
I stand corrected, and it makes sense if it only refers to the synthesized one.